### DISCUSSION TITLE
The same link traceId will have multiple similar trace data？

### DISCUSSION THREAD

**Discussion by zhang-zhipeng** [apache/skywalking#10009] [Q&A] [answered]
It will cause the number of requests for a service to be miscalculated

**Comment by zhang-zhipeng** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/42615740/203456835-6e7d7a7d-449f-453f-9096-4fc0ca733c1b.png)

**Reply by zhang-zhipeng** [reply]
service_request_count = from(Service.*).count();    If I use this metric, it calculates twice as much as I expect

**Reply by wu-sheng** [reply]
It will for sure. If you don't want this, remove undertow plugin.

**Reply by zhang-zhipeng** [reply]
thank you
